src/app/components/SubjectHeading/SubjectHeadings.jsx

```diff
@@ -0,0 +1,128 @@
+/* globals document */
+import React from 'react';
+import { Link } from 'react-router';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+import SubjectHeadingsList from './SubjectHeadingsList';
+import appConfig from '../../../../appConfig';
+
+
+class SubjectHeadings extends React.Component {
```

Comment:
What about calling this file `SubjectHeadingsContainer` so the file names are not so similar